### DISCUSSION TITLE
invalid bin entry for package @babel/parser@7.22.11

### DISCUSSION THREAD

**Discussion by xuyf1001** [babel/babel#15899] [General]
invalid bin entry for package @babel/parser@7.22.11

**Comment by JLHwung** [comment]
The current `bin` entry looks good to me:

https://github.com/babel/babel/blob/e43d8e77cf88245e930055e70a42f46e2996bce6/packages/babel-parser/package.json#L44

Can you elaborate? Why do you think it is invalid?
